aws/ec2.go

```diff
@@ -19,8 +19,10 @@ const (
 	kubernetesCreatorValue     = "kube-ingress-aws-controller"
 	autoScalingGroupNameTag    = "aws:autoscaling:groupName"
 	runningState               = 16 // See https://github.com/aws/aws-sdk-go/blob/master/service/ec2/api.go, type InstanceState
+	stoppedState               = 80 // See https://github.com/aws/aws-sdk-go/blob/master/service/ec2/api.go, type InstanceState
```

Comment:
I don't see this used anywhere?

Reply:
This is used only in tests. But the constant is not testing specific, so I put it here.